README.md

```diff
@@ -1,49 +1,61 @@
-# Jujutsu VCS
+<div class="title-block" style="text-align: center;" align="center">
 
-![](https://img.shields.io/github/license/martinvonz/jj) ![](https://img.shields.io/github/v/release/martinvonz/jj) ![](https://img.shields.io/github/release-date/martinvonz/jj) ![](https://img.shields.io/crates/v/jj-cli)
-<br/>
-![](https://github.com/martinvonz/jj/workflows/build/badge.svg) ![](https://img.shields.io/codefactor/grade/github/martinvonz/jj/main) ![](https://img.shields.io/librariesio/github/martinvonz/jj)
+# Jujutsu &mdash; a version control system
 
-- [Disclaimer](#disclaimer)
-- [Introduction](#introduction)
-- [Getting started](#getting-started)
-- [Status](#status)
-- [Related work](#related-work)
+![](https://img.shields.io/github/license/martinvonz/jj)
+![](https://img.shields.io/github/v/release/martinvonz/jj)
+![](https://img.shields.io/github/release-date/martinvonz/jj)
+![](https://img.shields.io/crates/v/jj-cli)
+<br/> ![](https://github.com/martinvonz/jj/workflows/build/badge.svg)
+![](https://img.shields.io/codefactor/grade/github/martinvonz/jj/main)
+![](https://img.shields.io/librariesio/github/martinvonz/jj)
 
-## Disclaimer
+<strong>
+  <a href="https://martinvonz.github.io/jj">Homepage</a>
```

Comment:
I would get rid of the "Homepage" for now. We should definitely bring it back once there's an actual Homepage on the website, though.

Reply:
Why do you think we need to get rid of it? Does just calling it "Documentation" instead of "Homepage" address your concern?

Reply:
It points to the same place as "Installation".

Reply:
Realistically, I feel like we should just leave this so it doesn't have to be added back later. In all honesty, about 99% of the README as it stands (even after this change) is what should be on the homepage, so I think moving from the README to `docs/` next is probably a good follow up?

Reply:
I was trying to avoid having the same information in two separate places that would need to be kept in sync. I'm not sure whether exactly the same README would work inside `docs/`, since the links in the README point to `https://martinvonz.github.io/jj/latest/...` while on the website they should point to just `...`.

Another minor note is that if you move the README, the demos will need to move too. So, in any case, please wait for https://github.com/martinvonz/jj/pull/2231 to get merged.

I was imagining a bare-bones homepage with links to the github repo and a table of contents for the website. Alternatively, we could get rid of most of the README in the repo and change it to be a link to the website (where all the content of the current README would go). I lean against that. For one, I think that having a nice ad for `jj` inside the GitHub repo is more important than having that ad on the website.

Another issue with the website is that its URLs are somewhat temporary, since I hope it'll eventually move to its own domain. (We'll probably try to set up redirects when that happens, but it could be tricky)

There might be other options that I haven't considered.

Reply:
> Realistically, I feel like we should just leave this so it doesn't have to be added back later.

OK.

(Non-actionable thoughts) Ideally, someone would have the time to create a bare-bones homepage soon. I don't have it right now. Once it exists, it will only go to the `prerelease` section of the website until the next release, but we can update the `latest` section of the website manually.

Reply:
Actually, I did create a (very) basic homepage in https://github.com/martinvonz/jj/pull/2327.